lib/query-params/format-value.js

```diff
@@ -0,0 +1,26 @@
+const splitOnUnescapedCommas = require('../../client/lib/split-commas.js');
```

Comment:
I can't find a _unit_ test for split-commas.js, am I just not looking in the right place?